textproc/xchm/Portfile

```diff
@@ -19,18 +20,19 @@ long_description    A cross-platform GUI front-end to chmlib, xCHM can \
                     a searchable index, and search for words in the current \
                     page and the whole book.
 
-homepage            http://xchm.sourceforge.net/
-
-fetch.type          git
-git.url             git://git.code.sf.net/p/xchm/code
+checksums           rmd160  d4e7414d94a171cd6137519a25d3dd0e4208bec7 \
+                    sha256  bb422a2a6893541fa168d66559244449c4d94d74b30967b23b23ef83feabb3c1 \
+                    size    963115
 
 app.name            xCHM
 app.executable      xchm
 app.icon            ${worksrcpath}/art/xchm-128.png
 
 wxWidgets.use       wxWidgets-3.0
 
-depends_build       port:makeicns
+depends_build       port:gettext \
```

Comment:
I'm not sure it matters, but `depends_build` should be `depends_build-append` to not override portgroup dependencies.

Reply:
It does matter because the `app` PortGroup declares dependency on `makeicns`, so one dependency may go